chord_metadata_service/mohpackets/data/small_dataset/mockaroo_schemas/Program.schema.json

```diff
@@ -23,7 +23,7 @@
       "null_percentage": 0,
       "type": "Template",
       "value": null,
-      "formula": "[\"BIO-CAN\",\n\"INSPIRE\",\n\"POG\",\n\"IO-ALINES\",\n\"COMPASS\"][random(0,4)]\n"
+      "formula": "[\"SYNTHETIC-BIO-CAN\",\n\"SYNTHETIC-INSPIRE\",\n\"SYNTHETIC-POG\",\n\"SYNTHETIC-IO-ALINES\",\n\"SYNTHETIC-COMPASS\"][random(0,4)]\n"
```

Comment:
Do we need "SYNTHETIC-IO-ALINES\" and "SYNTHETIC-COMPASS\" in this line?

Reply:
yea, I'm making this change so we won't mistake fake data for real data later. The mock data looks very similar to the real one that it's hard to tell them apart